transport/transporters/rabbitmq/config.go

```diff
@@ -0,0 +1,74 @@
+/*
+  Copyright 2019 Nextdoor.com, Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+package rabbitmq
+
+import (
+	cli "gopkg.in/Nextdoor/cli.v1"
+	"gopkg.in/Nextdoor/cli.v1/altsrc"
+)
+
+const (
+	ConfVarExchangeName   = "exchange-name"
+	ConfVarUsername       = "rabbitmq-username"
+	ConfVarPassword       = "rabbitmq-password"
+	ConfVarHost           = "rabbitmq-host"
+	ConfVarVirtualHost    = "rabbitmq-vhost"
+	ConfVarWriteBatchSize = "rabbitmq-batch-size"
+)
+
+var Flags = []cli.Flag{
+	cli.StringFlag{
```

Comment:
Can you add in SSL support?

Reply:
Sure.

Reply:
In-depth TLS support is a little more difficult (see example: https://godoc.org/github.com/streadway/amqp#DialTLS), but just turning it on should be as easy as setting the schema to `amqps`.